diff -up glib-networking-2.28.6.1/tls/gnutls/gtlsconnection-gnutls.c.0003-rh1176719-priority-string glib-networking-2.28.6.1/tls/gnutls/gtlsconnection-gnutls.c
--- glib-networking-2.28.6.1/tls/gnutls/gtlsconnection-gnutls.c.0003-rh1176719-priority-string	2016-05-05 15:50:44.210729383 +0200
+++ glib-networking-2.28.6.1/tls/gnutls/gtlsconnection-gnutls.c	2016-05-05 15:55:55.176673996 +0200
@@ -189,16 +189,16 @@ g_tls_connection_gnutls_init_priorities
   /* First field is "ssl3 only", second is "allow unsafe rehandshaking" */
 
   gnutls_priority_init (&priorities[FALSE][FALSE],
-			"NORMAL",
+			"NORMAL:%COMPAT:%LATEST_RECORD_VERSION",
 			NULL);
   gnutls_priority_init (&priorities[TRUE][FALSE],
-			"NORMAL:!VERS-TLS1.2:!VERS-TLS1.1:!VERS-TLS1.0",
+			"NORMAL:%COMPAT:%LATEST_RECORD_VERSION:!VERS-TLS1.2:!VERS-TLS1.1:!VERS-TLS1.0",
 			NULL);
   gnutls_priority_init (&priorities[FALSE][TRUE],
-			"NORMAL:%UNSAFE_RENEGOTIATION",
+			"NORMAL:%COMPAT:%LATEST_RECORD_VERSION:%UNSAFE_RENEGOTIATION",
 			NULL);
   gnutls_priority_init (&priorities[TRUE][TRUE],
-			"NORMAL:!VERS-TLS1.2:!VERS-TLS1.1:!VERS-TLS1.0:%UNSAFE_RENEGOTIATION",
+			"NORMAL:%COMPAT:%LATEST_RECORD_VERSION:!VERS-TLS1.2:!VERS-TLS1.1:!VERS-TLS1.0:%UNSAFE_RENEGOTIATION",
 			NULL);
 }
 
