diff -up glib-networking-2.28.6.1/configure.ac.no-gnome-proxy glib-networking-2.28.6.1/configure.ac
--- glib-networking-2.28.6.1/configure.ac.no-gnome-proxy	2014-06-09 11:04:41.812433092 -0400
+++ glib-networking-2.28.6.1/configure.ac	2014-06-09 11:04:58.830434014 -0400
@@ -66,7 +66,7 @@ dnl *** Checks for GNOME proxy backend *
 dnl **************************************
 AC_ARG_WITH(gnome-proxy,
     [AC_HELP_STRING([--with-gnome-proxy],
-                    [support for GNOME proxy configuration @<:@default=check@:>@])]
+                    [support for GNOME proxy configuration @<:@default=check@:>@])],
     [],
     [with_gnome_proxy=check])
 AS_IF([test "x$with_gnome_proxy" != "xno"],
diff -up glib-networking-2.28.6.1/configure.no-gnome-proxy glib-networking-2.28.6.1/configure
--- glib-networking-2.28.6.1/configure.no-gnome-proxy	2014-06-09 11:04:45.955433317 -0400
+++ glib-networking-2.28.6.1/configure	2014-06-09 11:04:55.021433808 -0400
@@ -13407,7 +13407,9 @@ fi
 
 # Check whether --with-gnome-proxy was given.
 if test "${with_gnome_proxy+set}" = set; then :
-  withval=$with_gnome_proxy; with_gnome_proxy=check
+  withval=$with_gnome_proxy;
+else
+  with_gnome_proxy=check
 fi
 
 if test "x$with_gnome_proxy" != "xno"; then :
